pkg/output/time-entry/markdown.gotmpl.md

```diff
@@ -23,21 +23,37 @@
   {{- $tags = "No Tags" -}}
 {{- end -}}
 
-{{- $pad := maxLength .Description $project $tags $bil -}}
+{{- $customFields := "" -}}
+{{- $hasCustomFields := false -}}
+{{- with .CustomFields -}}
+  {{- range $index, $element := . -}}
+    {{- $value := $element.ValueAsString -}}
+    {{- if ne $value "" -}}
+      {{- if ne $index 0 }}{{ $customFields = concat $customFields ", " }}{{ end -}}
+      {{- $customFields = concat $customFields $element.Name ": " $value -}}
+      {{- $hasCustomFields = true -}}
+    {{- end -}}
+  {{- end -}}
+{{- end -}}
+
+{{- $pad := maxLength .Description $project $tags $customFields $bil -}}
 
 ## _Time Entry_: {{ .ID }}
 
-_Time and date_  
+_Time and date_
```

Comment:
these are necessary to markdown interpreters to know that the line break is intensional 
```suggestion
_Time and date_  
```